ykcompile/src/lib.rs

```diff
@@ -27,6 +27,8 @@ pub enum CompileError {
     /// We ran out of registers.
     /// In the long-run, when we have a proper register allocator, this won't be needed.
     OutOfRegisters,
+    /// We tried to retrieve the register for a local which doesn't have a register assigned to it.
+    UnknownLocal,
```

Comment:
Shall we put the Local inside and report it when we crash?

Reply:
I just realised this isn't even used anymore, so I removed it in 
61e680e.